samples/nrf_desktop/src/hw_interface/motion_optical.c

```diff
@@ -1010,11 +1073,31 @@ static bool event_handler(const struct event_header *eh)
 
 	if (IS_ENABLED(CONFIG_DESKTOP_CONFIG_CHANNEL_ENABLE)) {
 		if (is_config_event(eh)) {
-			const struct config_event *event =
-				cast_config_event(eh);
+			const struct config_event *event = cast_config_event(eh);
 
-			update_config(event->id, event->dyndata.data,
+			if (is_my_config_id(event->id)) {
+				update_config(event->id, event->dyndata.data,
 				      event->dyndata.size);
+			}
+
+			return false;
+		}
+
+		if (is_config_fetch_request_event(eh)) {
+			const struct config_fetch_request_event *event =
+				cast_config_fetch_request_event(eh);
+
+			if (is_my_config_id(event->id)) {
+				size_t data_size = get_config_data_size(event->id);
+				struct config_fetch_event *fetch_event =
+					new_config_fetch_event(data_size);
+
+				fetch_event->id = event->id;
+				fetch_config(fetch_event->id,
```

Comment:
Initialized to predefined error value in case of fetching unsupported sensor option ?

Reply:
done

Reply:
I think what @MarekPieta meant is that there is no way of telling the other board that there was an error when fetching the data.